### ISSUE TITLE
More robust smoke test that will not randomly fail because of selenium time out

### ISSUE THREAD

**Issue by unkcpz** [aiidalab/aiidalab-qe#380]
The smoke tests failed randomly because the selenium call is not completed. We can increase the time to make sure the page is loaded, or a more clever detect for the successfully load of the page.

**Comment by danielhollas** [comment]
@unkcpz @yakutovicha I think I now have a pretty robust approach to ensure the page is loaded by checking that the app-busy appmode icon is invisible. Would be good to add this to both QeApp and AWB to reduce the test flakiness.

https://github.com/ispg-group/aiidalab-ispg/blob/7bbff87cb1576d43aca8921cfd34825963f3e159/tests/conftest.py#L87

**Comment by unkcpz** [comment]
Okay, thanks! I'll take a look and use it.

**Comment by danielhollas** [comment]
Yes, it's the gears logo, there's no magic in it, I just wait until the gears logo disappears.
